### PR TITLE
Add custom log & AppDynamics variables

### PR DESCRIPTION
**Pull request by apfeiffer85**
https://github.com/zalando-stups/taupage/issues/229

### CODE REVIEW

**Comment by apfeiffer85** [comment]
👍

File: components/taupage.rst [hunk 1]
```diff
@@ -556,12 +563,20 @@ scalyr_application_log_parser
...
+
+If the AppDynamics Agent is integrated in Taupage you can enable AppDyanmics with this variable and set your AppDynamics ApplicationName.
+
+appdynamics_machineagent_tiername
+---------------------------------
 
 **(optional)**
 
-If you provide the NewRelic license key in the .yaml file, the newrelic-sysmond will be set up.
+If you want to use Log-shipping without a App-Agent from AppDynamics you have to set the Tiername for the MachineAgent manually with this variable.
```
Comment (hjacobs) [inline components/taupage.rst]: - "log shipping"
- "without an"

**Comment by mrandi** [comment]
👍

**Comment by mrandi** [comment]
👍

**Comment by apfeiffer85** [comment]
👍

**Comment by apfeiffer85** [comment]
👍

**Pull request merged**